metalbear-co/sccache-action
# Get latest release and verify SHA-256 hash

- Get latest release with GitHub API (closes #4)
  - Fall back to known stable version if latest is not usable
  - Use `${{ secrets.GITHUB_TOKEN }}` to avoid issues with rate limiting
- Verify `sccache` SHA-256 hash (closes #5)
